frontend/src/metabase/query_builder/components/view/sidebars/QuestionInfoSidebar.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { PLUGIN_MODERATION, PLUGIN_MODEL_PERSISTENCE } from "metabase/plugins";
+
+import EditableText from "../../EditableText";
+import QuestionActivityTimeline from "metabase/query_builder/components/QuestionActivityTimeline";
+import Question from "metabase-lib/lib/Question";
+import { Card } from "metabase-types/types/Card";
+
+import { Root, ContentSection } from "./QuestionInfoSidebar.styled";
+
+interface Props {
+  question: Question;
+  onSave: (card: Card) => void;
+}
+
+export const QuestionInfoSidebar = ({ question, onSave }: Props) => {
+  const description = question.description();
+  const isDataset = question.isDataset();
+
+  const handleSave = (description: string) => {
+    if (question.description() !== description) {
+      onSave({
+        ...question.card(),
+        description,
+      });
+    }
+  };
+
+  return (
+    <Root>
+      <ContentSection>
+        <EditableText value={description} onChange={handleSave} />
+        <PLUGIN_MODERATION.QuestionModerationSection question={question} />
+      </ContentSection>
+
+      {isDataset && question.isPersisted() && (
```

Comment:
Nit: `isDataset` is defined at line 19, but `question.isPersisted()` is inlined. I wish they're more consistent.

Reply:
+1